syftlib/src/main/java/org/openmined/syft/proto/SyftModel.kt

```diff
@@ -7,6 +7,13 @@ import java.io.File
 
 private const val TAG = "SyftModel"
 
+/**
+ * SyftModel is a model class for neural network weights.
+ * kind of like nn.Module.state_dict
```

Comment:
Instead of kind of like----
Maybe go for something more formal SyftModel is the data model class for storing the weights of the neural network used for training or inference.
Also add the field descriptions for modelName, version, pyGridModelId, etc